components/tor/tor_control.h

```diff
@@ -155,15 +138,7 @@ class TorControl {
 
   TorControl::Delegate* delegate_;
 
-  void StartWatching();
-  void StopWatching();
-  void CheckingOldTorProcess(base::OnceClosure callback);
-  void WatchDirChanged(const base::FilePath& path, bool error);
-  void Poll();
-  void PollDone();
-  bool EatControlCookie(std::vector<uint8_t>&, base::Time&);
-  bool EatControlPort(int&, base::Time&);
-  bool EatOldPid(base::ProcessId* id);
+  base::WeakPtrFactory<TorControl> weak_ptr_factory_{this};
```

Comment:
could you please move all private fields below private methods, to make clear that `weak_ptr_factory_` is the last field in the list

Reply:
fixed in aeed04b506e574973c5df12c9781dde63e187342